src/hooks/useIsIosDevice.ts

```diff
@@ -0,0 +1,21 @@
+import { useState } from 'react';
+import { useInitial } from './useInitial';
+
+/**
+ * Hook for checking if page is loaded from iOS mobile device.
+ * @returns boolean true if on mobile.
+ */
+
+export function useIsIosDevice(): boolean {
```

Comment:
why is this even a hook? there is no reason for it to be a hook, I should just be a utils method
(unless im missing something important here 😅 )

Reply:
Hah, good question! I just moved it as is; but I agree: it doesn't seem that it should be a hook, but I'll double check.

Reply:
Update: ah, it's a hook because it uses `useInitial()` hook to check this only once per mount.
Makes sense, since the device shouldn't change so it should be enough to check it only once.